perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json

```diff
@@ -13,7 +13,8 @@
         "file-cache": {
           "max-size-mb": -1,
           "cache-file-for-range-read": true
-        }
+        },
+        "cache-dir": "/tmp/cache"
```

Comment:
With this change you are not creating new configuration, so although it will not throw an error and run with these configs only but on big query table it will not reflect this parameter. If you want to do so, you have to create new configuration(by giving different config_name)

Reply:
Done, changed the name.